src/test/java/com/celements/model/classes/fields/ClassFieldTest.java

```diff
@@ -28,7 +30,7 @@
   public void setUp() throws Exception {
     super.setUp();
```

Comment:
please do not override the default setUp method. Instead add a different named method with Before annotation.